### DISCUSSION TITLE
Error on demopage (MudExSelect)

### DISCUSSION THREAD

**Discussion by marcusjm** [fgilde/MudBlazor.Extensions#104] [General]
Seems Blazor gives an Error here. I didn't want to add it to issues since this is more about the Demo Page.
Any fix on the horizon?

Also. The DialogEx gives me some flickering when opening first time, this does not happen in the demo part. Any reason for this?

**Comment by fgilde** [comment]
Dont know about the error. I will have a Look into it.

For your flickering Problem, this can happen when using auto css loading and the additionally when then the dialog is the first thing that needs the css. To fix this you can also load the styles manually by adding the following line to your index.html or _Host.cshtml

`<link id="mudex-styles" href="_content/MudBlazor.Extensions/mudBlazorExtensions.min.css" rel="stylesheet">`

If you have loaded styles manually you should disable the automatic loading of the styles in the AddMudExtensions or AddMudServicesWithExtensions method. You can do this by adding the following line to your Startup.cs in the ConfigureServices method.

`builder.Services.AddMudServicesWithExtensions(c => c.WithoutAutomaticCssLoading());`

**Reply by marcusjm** [reply]
Ah thanx. I will try this. As for the error I do not know the cause but I couldn't get any of the MudSelects to work.

**Reply by marcusjm** [reply]
> Dont know about the error. I will have a Look into it.
> 
> For your flickering Problem, this can Happen when using auto css losging ans the Dialog is the First thing that needs the css. To fix this you can also load the styles manually by adding the following line to your index.html or _Host.cshtml
> 
> `<link id="mudex-styles" href="_content/MudBlazor.Extensions/mudBlazorExtensions.min.css" rel="stylesheet">`
> 
> If you have loaded styles manually you should disable the automatic loading of the styles in the AddMudExtensions or AddMudServicesWithExtensions method. You can do this by adding the following line to your Startup.cs in the ConfigureServices method.
> 
> `builder.Services.AddMudServicesWithExtensions(c => c.WithoutAutomaticCssLoading());`

Just to be clear. This page

https://mudblazor-extensions.pages.dev/mud-ex-select

**Reply by fgilde** [reply]
sorry for late response, currently I having mostly no time. The bug you mentioned is only in my deployed azure stack. I have runtime issues that I currently not find, but try still to fix that. However if you use the component in your project it should work. My local apps also works and another deployed app that uses this component also works

**Reply by fgilde** [reply]
fixed https://mudex.azurewebsites.net/mud-ex-select
